config/300-controller.yaml

```diff
@@ -24,15 +24,15 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: 3scale-kourier-control
+      app: net-kourier-controller
   template:
     metadata:
       labels:
-        app: 3scale-kourier-control
+        app: net-kourier-controller
     spec:
       containers:
         - image: ko://knative.dev/net-kourier/cmd/kourier
-          name: kourier-control
+          name: controller
```

Comment:
I'm fine for this change, but users have to change `spec.registry.override` if they configured :sweat_drops: . In addition, they need to change it `deployment/container` style like  https://github.com/knative/docs/pull/3821/files